### ISSUE TITLE
washingtonpost.com

### ISSUE THREAD

**Issue by maximevdl** [uBlockOrigin/uAssets#7476]
### URL(s) where the issue occurs
`
https://www.washingtonpost.com/politics/trump-says-torture-works-backs-waterboarding-and-much-worse/2016/02/17/4c9277be-d59c-11e5-b195-2e29a4e13425_story.html`

### Describe the issue

Anti-adblock

### Screenshot(s)

![1](https://user-images.githubusercontent.com/33755812/83248084-ce88c600-a1a4-11ea-8a65-5f39a4825c40.PNG)
![2](https://user-images.githubusercontent.com/33755812/83248101-d6486a80-a1a4-11ea-8745-6dfd4cd47d62.PNG)

### Versions

- Browser/version: Chromium v83.0.4103.61
- uBlock Origin version: v1.27.11b0

### Settings

- Default

**Comment by Yuki2718** [comment]
Can't reproduce on my end.

**Comment by llacb47** [comment]
Update filters?

**Comment by maximevdl** [comment]
> Update filters?

They were up to date.

**Comment by llacb47** [comment]
can you add these filters: 
```
washingtonpost.com##+js(nostif)
washingtonpost.com##+js(nosiif)
```

Then open the developer console, refresh the page, and copy everything into a comment.

**Comment by uBlock-user** [comment]
@maximevdl Does `washingtonpost.com##+js(nostif, _0x)` work for you on Chrome ?

**Comment by maximevdl** [comment]
> can you add these filters:
> 
> ```
> washingtonpost.com##+js(nostif)
> washingtonpost.com##+js(nosiif)
> ```
> 
> Then open the developer console, refresh the page, and copy everything logged into the console into a new comment.

There's a lot of content in the console, I'm afraid it's gonna be unreadable if I copy and paste everything here.

> @maximevdl Does `washingtonpost.com##+js(nostif, _0x)` work for you on Chrome ?

Unfortunately it doesn't.

You both weren't able to reproduce either ?

**Comment by llacb47** [comment]
@maximevdl it won't be unreadable. Please copy and paste it all or take a screenshot.

**Comment by uBlock-user** [comment]
I was able to reproduce on Firefox(intermittently), for that I pushed https://github.com/uBlockOrigin/uAssets/commit/4287f12e4d89249aebe2b8685b276ea4015504f4, but I'm not on Chromium/Chrome.

**Comment by okiehsch** [comment]
I can't reproduce using Chromium/Chrome or Edge.

**Comment by mapx-** [comment]
@maximevdl are you using some other blocking extension alongside uBO ?

**Comment by maximevdl** [comment]
> @maximevdl it won't be unreadable. Please copy and paste it all or take a screenshot.

VM406:117 uBO: setTimeout("function z(){K()}", 0)
4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:120 GET https://www.washingtonpost.com/wp-stat/ad/zeus/wp-ad.min.js net::ERR_BLOCKED_BY_CLIENT
4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:127 GET https://c.go-mpulse.net/boomerang/W8234-EWWKH-SQWJU-EAC6K-7AE5Z net::ERR_BLOCKED_BY_CLIENT
doc.open._l @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:127
onload @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:1
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:128
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:128
load (async)
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:128
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:128
VM406:117 uBO: setTimeout("function(){_0x3fc8ea({"type":"ex","result":![],"overlay":![]})}", 8000)
VM406:117 uBO: setTimeout("function(){for(var i=0;i<unset.length;i++){document.cookie=unset[i]+"\x3d; expires\x3dThu, 01 Jan 1970 00:00:01 GMT; path\x3d/; domain\x3dwww.washingtonpost.com";document.cookie=unset[i]+"\x3d; expires\x3dThu, 01 Jan 1970 00:00:01 GMT; path\x3d/; domain\x3d.washingtonpost.com";document.cookie=unset[i]+"\x3d; expires\x3dThu, 01 Jan 1970 00:00:01 GMT; path\x3d/; domain\x3d"}}", 0)
VM406:117 uBO: setTimeout("function(wait){if(wait===true?--jQuery.readyWait:jQuery.isReady)return;jQuery.isReady=true;if(wait!==true&&--jQuery.readyWait>
0)return;readyList.resolveWith(document,[jQuery])}", undefined)
VM406:117 uBO: setTimeout("function(){try{mightThrow()}catch(e){if(jQuery.Deferred.exceptionHook)jQuery.Deferred.exceptionHook(e,
process.stackTrace);if(depth+1>=maxDepth){if(handler!==Thrower){that=undefined;args=[e]}deferred.rejectWith(that,args)}}}", undefined)
VM406:117 uBO: setTimeout("function(){try{mightThrow()}catch(e){if(jQuery.Deferred.exceptionHook)jQuery.Deferred.exceptionHook(e,
process.stackTrace);if(depth+1>=maxDepth){if(handler!==Thrower){that=undefined;args=[e]}deferred.rejectWith(that,args)}}}", undefined)
6b62210598.js?_=25816:9 TWP.Identity
6b62210598.js?_=25816:41 [0.15] STARTING Paywall
6b62210598.js?_=25816:41 PAYWALL CHECK STATUS: 
6b62210598.js?_=25816:41 PWAPI URL: https://pwapi.washingtonpost.com/politics/trump-says-torture-works-backs-waterboarding-and-much-worse/2016/02/17/4c9277be-d59c-11e5-b195-2e29a4e13425_story.html?rplapisplit=1&referrer=https://www.washingtonpost.com/politics/trump-says-torture-works-backs-waterboarding-and-much-worse/2016/02/17/4c9277be-d59c-11e5-b195-2e29a4e13425_story.html
VM406:117 uBO: setTimeout("function(){try{mightThrow()}catch(e){if(jQuery.Deferred.exceptionHook)jQuery.Deferred.exceptionHook(e,
process.stackTrace);if(depth+1>=maxDepth){if(handler!==Thrower){that=undefined;args=[e]}deferred.rejectWith(that,args)}}}", undefined)
VM406:117 uBO: setTimeout("function(){try{mightThrow()}catch(e){if(jQuery.Deferred.exceptionHook)jQuery.Deferred.exceptionHook(e,
process.stackTrace);if(depth+1>=maxDepth){if(handler!==Thrower){that=undefined;args=[e]}deferred.rejectWith(that,args)}}}", undefined)
VM406:117 uBO: setTimeout("function(){try{mightThrow()}catch(e){if(jQuery.Deferred.exceptionHook)jQuery.Deferred.exceptionHook(e,
process.stackTrace);if(depth+1>=maxDepth){if(handler!==Thrower){that=undefined;args=[e]}deferred.rejectWith(that,args)}}}", undefined)
An unknown error occurred when fetching the script.
VM406:117 uBO: setTimeout("function(){try{mightThrow()}catch(e){if(jQuery.Deferred.exceptionHook)jQuery.Deferred.exceptionHook(e,
process.stackTrace);if(depth+1>=maxDepth){if(handler!==Thrower){that=undefined;args=[e]}deferred.rejectWith(that,args)}}}", undefined)
VM406:117 uBO: setTimeout("function(){try{mightThrow()}catch(e){if(jQuery.Deferred.exceptionHook)jQuery.Deferred.exceptionHook(e,
process.stackTrace);if(depth+1>=maxDepth){if(handler!==Thrower){that=undefined;args=[e]}deferred.rejectWith(that,args)}}}", undefined)
VM406:117 uBO: setTimeout("function(){if(!store.isFired&&"function"==typeof window._dw)_dw.push(["global","ready",function(api){store.isFired=true;fireEvent(api)}])}", 200)
VM406:117 uBO: setTimeout("function(){try{mightThrow()}catch(e){if(jQuery.Deferred.exceptionHook)jQuery.Deferred.exceptionHook(e,
process.stackTrace);if(depth+1>=maxDepth){if(handler!==Thrower){that=undefined;args=[e]}deferred.rejectWith(that,args)}}}", undefined)
VM406:117 uBO: setTimeout("function () { [native code] }", 3000)
VM406:117 uBO: setTimeout("function(){try{mightThrow()}catch(e){if(jQuery.Deferred.exceptionHook)jQuery.Deferred.exceptionHook(e,
process.stackTrace);if(depth+1>=maxDepth){if(handler!==Thrower){that=undefined;args=[e]}deferred.rejectWith(that,args)}}}", undefined)
VM406:117 uBO: setTimeout("function(){m(e,function e(t,n){return null===t?(c(),0):(n||/^https?:\/\//.test(t)||!p||(t=-1===t.indexOf(".js")?p+t+".js":p+t),v[t]?2==v[t]?c():setTimeout(function(){e(t,!0)},0):(v[t]=1,void y(t,c)))})}", 0)
VM406:117 uBO: setTimeout("function i(){s.copyTWPToWd(),s.initForTwp(),y.wp_events=y.wp_events?y.wp_events:"",y.wp_events=y.wp_suppressDefaultEvent?y.wp_events:$e(y.wp_defaultEvents,y.wp_events),y.s_code=s.t();We(s);u&&l(s),ze.setReady()}", 0)
2VM406:117 uBO: setTimeout("function(){try{mightThrow()}catch(e){if(jQuery.Deferred.exceptionHook)jQuery.Deferred.exceptionHook(e,
process.stackTrace);if(depth+1>=maxDepth){if(handler!==Thrower){that=undefined;args=[e]}deferred.rejectWith(that,args)}}}", undefined)
2VM406:117 uBO: setTimeout("function(){try{mightThrow()}catch(e){if(jQuery.Deferred.exceptionHook)jQuery.Deferred.exceptionHook(e,
process.stackTrace);if(depth+1>=maxDepth){if(handler!==Thrower){that=undefined;args=[e]}deferred.rejectWith(that,args)}}}", undefined)
VM406:117 uBO: setTimeout("function(){try{mightThrow()}catch(e){if(jQuery.Deferred.exceptionHook)jQuery.Deferred.exceptionHook(e,
process.stackTrace);if(depth+1>=maxDepth){if(handler!==Thrower){that=undefined;args=[e]}deferred.rejectWith(that,args)}}}", undefined)
VM406:117 uBO: setTimeout("function(){$closeBtn.addClass("fa-remove")}", 6000)
VM406:117 uBO: setTimeout("function(){try{mightThrow()}catch(e){if(jQuery.Deferred.exceptionHook)jQuery.Deferred.exceptionHook(e,
process.stackTrace);if(depth+1>=maxDepth){if(handler!==Thrower){that=undefined;args=[e]}deferred.rejectWith(that,args)}}}", undefined)
VM406:117 uBO: setTimeout("function(){try{mightThrow()}catch(e){if(jQuery.Deferred.exceptionHook)jQuery.Deferred.exceptionHook(e,
process.stackTrace);if(depth+1>=maxDepth){if(handler!==Thrower){that=undefined;args=[e]}deferred.rejectWith(that,args)}}}", undefined)
VM406:117 uBO: setTimeout("function(){try{mightThrow()}catch(e){if(jQuery.Deferred.exceptionHook)jQuery.Deferred.exceptionHook(e,
process.stackTrace);if(depth+1>=maxDepth){if(handler!==Thrower){that=undefined;args=[e]}deferred.rejectWith(that,args)}}}", undefined)
VM406:117 uBO: setTimeout("function(){t.F&&(t.complete?t.na():(f.trackOffline&&t.abort&&t.abort(),t.Ca()))}", 5000)
VM406:117 uBO: setTimeout("function(){return e(n)}", 10000)
VM406:117 uBO: setTimeout("function(){return e(n)}", 10000)
VM406:117 uBO: setTimeout("function(){return e(n)}", 3000)
imrs.php%3Fsrc%3D%26amp%3Bw%3D1440:1 GET https://www.washingtonpost.com/resizer/kYmJQMFewokX14LhsK01vB9TGGw=/60x60/www.washingtonpost.com/wp-apps/imrs.php%3Fsrc%3D%26amp%3Bw%3D1440 404
Image (async)
attr @ 0cbb1eb98e.js?_=0bc19:197
access @ 0cbb1eb98e.js?_=0bc19:101
attr @ 0cbb1eb98e.js?_=0bc19:196
(anonymous) @ 0cbb1eb98e.js?_=0bc19:376
each @ 0cbb1eb98e.js?_=0bc19:8
each @ 0cbb1eb98e.js?_=0bc19:3
wp_pb.StaticMethods.swapImage @ 0cbb1eb98e.js?_=0bc19:375
swapImage @ e7f066f51e.js?_=cd70e:1
init @ e7f066f51e.js?_=cd70e:1
(anonymous) @ e7f066f51e.js?_=cd70e:2
mightThrow @ 0cbb1eb98e.js?_=0bc19:93
process @ 0cbb1eb98e.js?_=0bc19:94
setTimeout (async)
apply @ VM406:42
apply @ VM406:128
(anonymous) @ 0cbb1eb98e.js?_=0bc19:95
fire @ 0cbb1eb98e.js?_=0bc19:87
add @ 0cbb1eb98e.js?_=0bc19:88
(anonymous) @ 0cbb1eb98e.js?_=0bc19:95
Deferred @ 0cbb1eb98e.js?_=0bc19:97
then @ 0cbb1eb98e.js?_=0bc19:95
jQuery.fn.ready @ 0cbb1eb98e.js?_=0bc19:99
(anonymous) @ e7f066f51e.js?_=cd70e:2
(anonymous) @ e7f066f51e.js?_=cd70e:3
4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:1 Uncaught (in promise) TypeError: Failed to register a ServiceWorker for scope ('https://www.washingtonpost.com/') with script ('https://www.washingtonpost.com/notification-sw.js'): An unknown error occurred when fetching the script.
VM406:117 uBO: setTimeout("function(){try{mightThrow()}catch(e){if(jQuery.Deferred.exceptionHook)jQuery.Deferred.exceptionHook(e,
process.stackTrace);if(depth+1>=maxDepth){if(handler!==Thrower){that=undefined;args=[e]}deferred.rejectWith(that,args)}}}", undefined)
VM406:117 uBO: setTimeout("function(){try{mightThrow()}catch(e){if(jQuery.Deferred.exceptionHook)jQuery.Deferred.exceptionHook(e,
process.stackTrace);if(depth+1>=maxDepth){if(handler!==Thrower){that=undefined;args=[e]}deferred.rejectWith(that,args)}}}", undefined)
VM406:117 uBO: setTimeout("function(){try{mightThrow()}catch(e){if(jQuery.Deferred.exceptionHook)jQuery.Deferred.exceptionHook(e,
process.stackTrace);if(depth+1>=maxDepth){if(handler!==Thrower){that=undefined;args=[e]}deferred.rejectWith(that,args)}}}", undefined)
VM406:117 uBO: setTimeout("function(){try{mightThrow()}catch(e){if(jQuery.Deferred.exceptionHook)jQuery.Deferred.exceptionHook(e,
process.stackTrace);if(depth+1>=maxDepth){if(handler!==Thrower){that=undefined;args=[e]}deferred.rejectWith(that,args)}}}", undefined)
6VM406:117 uBO: setTimeout("function(){try{mightThrow()}catch(e){if(jQuery.Deferred.exceptionHook)jQuery.Deferred.exceptionHook(e,
process.stackTrace);if(depth+1>=maxDepth){if(handler!==Thrower){that=undefined;args=[e]}deferred.rejectWith(that,args)}}}", undefined)
VM406:117 uBO: setTimeout("function(){try{mightThrow()}catch(e){if(jQuery.Deferred.exceptionHook)jQuery.Deferred.exceptionHook(e,
process.stackTrace);if(depth+1>=maxDepth){if(handler!==Thrower){that=undefined;args=[e]}deferred.rejectWith(that,args)}}}", undefined)
main.js?token=202005271849EST:1 GET https://static.chartbeat.com/js/chartbeat.js net::ERR_BLOCKED_BY_CLIENT
y @ main.js?token=202005271849EST:1
e @ main.js?token=202005271849EST:1
(anonymous) @ main.js?token=202005271849EST:1
g @ main.js?token=202005271849EST:1
m @ main.js?token=202005271849EST:1
(anonymous) @ main.js?token=202005271849EST:1
setTimeout (async)
apply @ VM406:42
apply @ VM406:128
h @ main.js?token=202005271849EST:1
(anonymous) @ main.js?token=202005271849EST:1
w @ main.js?token=202005271849EST:1
Rt @ main.js?token=202005271849EST:1
Nt @ main.js?token=202005271849EST:1
ln.Analytics.init @ main.js?token=202005271849EST:1
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:186
fire @ 0cbb1eb98e.js?_=0bc19:87
fireWith @ 0cbb1eb98e.js?_=0bc19:90
mightThrow @ 0cbb1eb98e.js?_=0bc19:94
process @ 0cbb1eb98e.js?_=0bc19:94
setTimeout (async)
apply @ VM406:42
apply @ VM406:128
(anonymous) @ 0cbb1eb98e.js?_=0bc19:95
fire @ 0cbb1eb98e.js?_=0bc19:87
fireWith @ 0cbb1eb98e.js?_=0bc19:90
fire @ 0cbb1eb98e.js?_=0bc19:90
fire @ 0cbb1eb98e.js?_=0bc19:87
fireWith @ 0cbb1eb98e.js?_=0bc19:90
deferred.<computed> @ 0cbb1eb98e.js?_=0bc19:96
fire @ 0cbb1eb98e.js?_=0bc19:87
fireWith @ 0cbb1eb98e.js?_=0bc19:90
deferred.<computed> @ 0cbb1eb98e.js?_=0bc19:96
(anonymous) @ 0cbb1eb98e.js?_=0bc19:265
n @ 0cbb1eb98e.js?_=0bc19:272
u @ 0cbb1eb98e.js?_=0bc19:272
(anonymous) @ 0cbb1eb98e.js?_=0bc19:290
e @ 0cbb1eb98e.js?_=0bc19:276
load (async)
c.downloadScriptElement @ 0cbb1eb98e.js?_=0bc19:276
(anonymous) @ 0cbb1eb98e.js?_=0bc19:290
each @ 0cbb1eb98e.js?_=0bc19:8
(anonymous) @ 0cbb1eb98e.js?_=0bc19:290
mightThrow @ 0cbb1eb98e.js?_=0bc19:93
process @ 0cbb1eb98e.js?_=0bc19:94
setTimeout (async)
apply @ VM406:42
apply @ VM406:128
(anonymous) @ 0cbb1eb98e.js?_=0bc19:95
fire @ 0cbb1eb98e.js?_=0bc19:87
add @ 0cbb1eb98e.js?_=0bc19:88
(anonymous) @ 0cbb1eb98e.js?_=0bc19:95
Deferred @ 0cbb1eb98e.js?_=0bc19:97
then @ 0cbb1eb98e.js?_=0bc19:95
f @ 0cbb1eb98e.js?_=0bc19:290
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:186
(anonymous) @ 0cbb1eb98e.js?_=0bc19:526
(anonymous) @ 0cbb1eb98e.js?_=0bc19:526
4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:230 GET https://securepubads.g.doubleclick.net/gampad/adx?iu=/701/ad-blk&sz=2x1&c=801568858309542&tile=1&u_tz=-120 net::ERR_BLOCKED_BY_CLIENT
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:230
_0x5ec10f @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:229
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:231
_0x40c4d7 @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:230
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:233
_0x33a23f @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:233
_0x803433 @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:234
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:241
Promise.then (async)
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:241
Promise.then (async)
_0x22524e @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:241
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:244
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:244
2VM406:117 uBO: setTimeout("function(){jqXHR.abort("timeout")}", 3600)
6b62210598.js?_=25816:42 PAYWALL ACTION: 6
6b62210598.js?_=25816:42 PAYWALL URL: (null)
6b62210598.js?_=25816:42 PAYWALL SUB: Unsubscribed
6b62210598.js?_=25816:42 PAYWALL METERED: Metered
6b62210598.js?_=25816:42 PAYWALL METER COUNT: 1
6b62210598.js?_=25816:43 PAYWALL MODE: 1
VM406:117 uBO: setTimeout("function(){_0x57d1b2(_0x3a188f)}", 1000)
VM406:117 uBO: setTimeout("function(){t.F&&(t.complete?t.na():(f.trackOffline&&t.abort&&t.abort(),t.Ca()))}", 5000)
4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:229 GET https://d2ty8gaf6rmowa.cloudfront.net/ad/ads.js net::ERR_BLOCKED_BY_CLIENT
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:229
_0xe0c497 @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:227
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:231
Promise.then (async)
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:231
_0x40c4d7 @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:230
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:233
_0x33a23f @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:233
_0x803433 @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:234
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:241
Promise.then (async)
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:241
Promise.then (async)
_0x22524e @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:241
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:244
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:244
VM406:117 uBO: setTimeout("function(){try{mightThrow()}catch(e){if(jQuery.Deferred.exceptionHook)jQuery.Deferred.exceptionHook(e,
process.stackTrace);if(depth+1>=maxDepth){if(handler!==Thrower){that=undefined;args=[e]}deferred.rejectWith(that,args)}}}", undefined)
VM406:117 uBO: setTimeout("function(){jqXHR.abort("timeout")}", 8000)
VM406:117 uBO: setTimeout("function(){try{mightThrow()}catch(e){if(jQuery.Deferred.exceptionHook)jQuery.Deferred.exceptionHook(e,
process.stackTrace);if(depth+1>=maxDepth){if(handler!==Thrower){that=undefined;args=[e]}deferred.rejectWith(that,args)}}}", undefined)
VM406:117 uBO: setTimeout("function(){try{mightThrow()}catch(e){if(jQuery.Deferred.exceptionHook)jQuery.Deferred.exceptionHook(e,
process.stackTrace);if(depth+1>=maxDepth){if(handler!==Thrower){that=undefined;args=[e]}deferred.rejectWith(that,args)}}}", undefined)
VM406:117 uBO: setTimeout("function(){__e._data[name]=true;for(var i=0;i<__e.length;i++)if(__e[i][0].toLowerCase()==name&&__e[i][1]){__e[i].__e_call=true;try{__e[i][1]()}catch(e){}}if(cb)cb()}", 0)
VM406:117 uBO: setTimeout("function(){__e._data[name]=true;for(var i=0;i<__e.length;i++)if(__e[i][0].toLowerCase()==name&&__e[i][1]){__e[i].__e_call=true;try{__e[i][1]()}catch(e){}}if(cb)cb()}", 0)
VM406:117 uBO: setTimeout("function a(){(s.api||s.feature)&&(d.aborted=!0,s=d.backlog={})}", 30000)
4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:255 GET https://js-agent.newrelic.com/nr-1118.min.js net::ERR_BLOCKED_BY_CLIENT
r @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:255
load (async)
__nr_require.loader @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:256
r @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:247
__nr_require.1 @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:247
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:247
0cbb1eb98e.js?_=0bc19:98 jQuery.Deferred exception: Cannot set property 'outerHTML' of null TypeError: Cannot set property 'outerHTML' of null
    at https://www.washingtonpost.com/pb/gr/c/default-article/r0zsCrj34ER7Wr/load_async_loader/bf6f3693c3.js?_=85209:9:263
    at Array.forEach (<anonymous>)
    at injectRenderings (https://www.washingtonpost.com/pb/gr/c/default-article/r0zsCrj34ER7Wr/load_async_loader/bf6f3693c3.js?_=85209:8:287)
    at https://www.washingtonpost.com/pb/gr/c/default-article/r0zsCrj34ER7Wr/load_async_loader/bf6f3693c3.js?_=85209:2:456
    at mightThrow (https://www.washingtonpost.com/pb/gr/ro/default-article/r0zsCrj34ER7Wr/load_immediately/0cbb1eb98e.js?_=0bc19:93:330)
    at process (https://www.washingtonpost.com/pb/gr/ro/default-article/r0zsCrj34ER7Wr/load_immediately/0cbb1eb98e.js?_=0bc19:94:485) undefined
jQuery.Deferred.exceptionHook @ 0cbb1eb98e.js?_=0bc19:98
process @ 0cbb1eb98e.js?_=0bc19:94
setTimeout (async)
apply @ VM406:42
apply @ VM406:128
(anonymous) @ 0cbb1eb98e.js?_=0bc19:95
fire @ 0cbb1eb98e.js?_=0bc19:87
fireWith @ 0cbb1eb98e.js?_=0bc19:90
fire @ 0cbb1eb98e.js?_=0bc19:90
fire @ 0cbb1eb98e.js?_=0bc19:87
fireWith @ 0cbb1eb98e.js?_=0bc19:90
mightThrow @ 0cbb1eb98e.js?_=0bc19:94
process @ 0cbb1eb98e.js?_=0bc19:94
setTimeout (async)
apply @ VM406:42
apply @ VM406:128
(anonymous) @ 0cbb1eb98e.js?_=0bc19:95
fire @ 0cbb1eb98e.js?_=0bc19:87
fireWith @ 0cbb1eb98e.js?_=0bc19:90
fire @ 0cbb1eb98e.js?_=0bc19:90
fire @ 0cbb1eb98e.js?_=0bc19:87
fireWith @ 0cbb1eb98e.js?_=0bc19:90
deferred.<computed> @ 0cbb1eb98e.js?_=0bc19:96
fire @ 0cbb1eb98e.js?_=0bc19:87
fireWith @ 0cbb1eb98e.js?_=0bc19:90
deferred.<computed> @ 0cbb1eb98e.js?_=0bc19:96
(anonymous) @ 0cbb1eb98e.js?_=0bc19:265
n @ 0cbb1eb98e.js?_=0bc19:272
u @ 0cbb1eb98e.js?_=0bc19:272
(anonymous) @ 0cbb1eb98e.js?_=0bc19:286
fire @ 0cbb1eb98e.js?_=0bc19:87
fireWith @ 0cbb1eb98e.js?_=0bc19:90
done @ 0cbb1eb98e.js?_=0bc19:240
(anonymous) @ 0cbb1eb98e.js?_=0bc19:246
load (async)
send @ 0cbb1eb98e.js?_=0bc19:246
ajax @ 0cbb1eb98e.js?_=0bc19:237
z @ 0cbb1eb98e.js?_=0bc19:273
(anonymous) @ 0cbb1eb98e.js?_=0bc19:286
(anonymous) @ 0cbb1eb98e.js?_=0bc19:289
each @ 0cbb1eb98e.js?_=0bc19:8
f @ 0cbb1eb98e.js?_=0bc19:289
injectResources @ bf6f3693c3.js?_=85209:7
injectAsyncFeatures @ bf6f3693c3.js?_=85209:2
(anonymous) @ bf6f3693c3.js?_=85209:1
xmlhttp.onreadystatechange @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:312
XMLHttpRequest.send (async)
window.pageBuilder._getJson @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:322
window.pageBuilder.loadAsyncFeatures @ bf6f3693c3.js?_=85209:1
window.pageBuilder.loadAsyncOverride @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:244
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:245
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:247
fire @ 0cbb1eb98e.js?_=0bc19:87
fireWith @ 0cbb1eb98e.js?_=0bc19:90
mightThrow @ 0cbb1eb98e.js?_=0bc19:94
process @ 0cbb1eb98e.js?_=0bc19:94
setTimeout (async)
apply @ VM406:42
apply @ VM406:128
(anonymous) @ 0cbb1eb98e.js?_=0bc19:95
fire @ 0cbb1eb98e.js?_=0bc19:87
fireWith @ 0cbb1eb98e.js?_=0bc19:90
fire @ 0cbb1eb98e.js?_=0bc19:90
fire @ 0cbb1eb98e.js?_=0bc19:87
fireWith @ 0cbb1eb98e.js?_=0bc19:90
deferred.<computed> @ 0cbb1eb98e.js?_=0bc19:96
fire @ 0cbb1eb98e.js?_=0bc19:87
fireWith @ 0cbb1eb98e.js?_=0bc19:90
deferred.<computed> @ 0cbb1eb98e.js?_=0bc19:96
(anonymous) @ 0cbb1eb98e.js?_=0bc19:265
n @ 0cbb1eb98e.js?_=0bc19:272
u @ 0cbb1eb98e.js?_=0bc19:272
(anonymous) @ 0cbb1eb98e.js?_=0bc19:290
e @ 0cbb1eb98e.js?_=0bc19:276
load (async)
c.downloadScriptElement @ 0cbb1eb98e.js?_=0bc19:276
(anonymous) @ 0cbb1eb98e.js?_=0bc19:290
each @ 0cbb1eb98e.js?_=0bc19:8
(anonymous) @ 0cbb1eb98e.js?_=0bc19:290
mightThrow @ 0cbb1eb98e.js?_=0bc19:93
process @ 0cbb1eb98e.js?_=0bc19:94
setTimeout (async)
apply @ VM406:42
apply @ VM406:128
(anonymous) @ 0cbb1eb98e.js?_=0bc19:95
fire @ 0cbb1eb98e.js?_=0bc19:87
add @ 0cbb1eb98e.js?_=0bc19:88
(anonymous) @ 0cbb1eb98e.js?_=0bc19:95
Deferred @ 0cbb1eb98e.js?_=0bc19:97
then @ 0cbb1eb98e.js?_=0bc19:95
f @ 0cbb1eb98e.js?_=0bc19:290
(anonymous) @ 4c9277be-d59c-11e5-b195-2e29a4e13425_story.html:247
(anonymous) @ 0cbb1eb98e.js?_=0bc19:526
(anonymous) @ 0cbb1eb98e.js?_=0bc19:526
2VM406:117 uBO: setTimeout("function(){__e._data[name]=true;for(var i=0;i<__e.length;i++)if(__e[i][0].toLowerCase()==name&&__e[i][1]){__e[i].__e_call=true;try{__e[i][1]()}catch(e){}}if(cb)cb()}", 0)
2VM406:117 uBO: setTimeout("function(){__e._data[name]=true;for(var i=0;i<__e.length;i++)if(__e[i][0].toLowerCase()==name&&__e[i][1]){__e[i].__e_call=true;try{__e[i][1]()}catch(e){}}if(cb)cb()}", 0)
2VM406:117 uBO: setTimeout("function(){__e._data[name]=true;for(var i=0;i<__e.length;i++)if(__e[i][0].toLowerCase()==name&&__e[i][1]){__e[i].__e_call=true;try{__e[i][1]()}catch(e){}}if(cb)cb()}", 0)
VM406:117 uBO: setTimeout("function(){t.F&&(t.complete?t.na():(f.trackOffline&&t.abort&&t.abort(),t.Ca()))}", 5000)

**Comment by maximevdl** [comment]
> @maximevdl are you using some other blocking extension alongside uBO ?

Nope, only using uBlock origin.

> I can't reproduce using Chromium/Chrome or Edge.

Adblocker also detected on my Edge v83.0.478.37 using uBlock origin v1.27.8 from the Microsoft Store.

**Comment by okiehsch** [comment]
> detected on my Edge v83.0.478.37 using uBlock origin v1.27.8 from the Microsoft Store. 

![image](https://user-images.githubusercontent.com/16838044/83324614-ba0d0200-a266-11ea-8748-90553664d236.png)

Edget on my end with a default setup and uBlock origin v1.27.

**Comment by Yuki2718** [comment]
No idea how to reproduce myself, but I found in some occasion current filter
`washingtonpost.com##div:has-text(We noticed you’re blocking ads.)`
doesn't match due to a variant of `’` and instead
`washingtonpost.com##div:has-text(We noticed you're blocking ads.)`
matches. The below shows current uBlock filter rule doesn't match.

![washingtonpost](https://user-images.githubusercontent.com/58900598/86924925-1c60f880-c16b-11ea-9e90-4f960b55ecc0.png)

This from logger is when the second rule matched, though now I can't reproduce.

![washingtonpost2](https://user-images.githubusercontent.com/58900598/86925031-44505c00-c16b-11ea-9dbe-0d8d207719a3.png)
